integrations/slack/events/app_home_opened.go

```diff
@@ -0,0 +1,39 @@
+package events
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"go.uber.org/zap"
+)
+
+
+type AppHomeOpen struct {
+    Type string `json:"type,omitempty"`
+    User string `json:"user,omitempty"`
+    Channel string `json:"channel,omitempty"`
+    Tab string `json:"tab,omitempty"`
+    EventTS string `json:"event_ts,omitempty"`
+}
+
+type appHomeOpenContainer struct {
+    Event *AppHomeOpen `json:"event"`
+}
+
+func AppHomeOpenHandler(l *zap.Logger, w http.ResponseWriter, body []byte, cb *Callback) any{
+    // Ignore self-triggered events.
+    for _, a := range cb.Authorizations {
+        if a.UserID == cb.Event.User {
+            l.Debug("Ignoring self-triggered event")
+            return nil
+        }
+    }
+
+    j := &appHomeOpenContainer{}
+    if err := json.Unmarshal(body, j); err != nil {
```

Comment:
```suggestion
    var j appHomeOpenContainer
    if err := json.Unmarshal(body, &j); err != nil {
```

Reply:
Personally, I try to minimize using `var`. It feels less explicit to me, but I admit it's just a personal stylistic choice.

@tebeka your suggestion works too, but can you explain why you think it's better? If it is, we could do a fix across all other events (in a separate PR).